docs/tutorial.md

```diff
@@ -8,53 +8,56 @@ To illustrate the concept of working with Group, Flavor, and Instance plugins, w
 All InfraKit plugins will by default open the unix socket located at /run/infrakit/plugins. Make sure this directory exists on your host:
 
 ```shell
-$ mkdir -p /run/infrakit/plugins/
-$ chmod 777 /run/infrakit/plugins
+# Make sure directory exists for plugins to discover each other (default is: /run/infrakit/plugins/)
+$ mkdir -p /tmp/infrakit/plugins/
```

Comment:
Not terribly important as this doc is only for broad guidance, but `$TMPDIR` is likely more portable than `/tmp`.